.licenserc.yaml

```diff
@@ -30,6 +30,7 @@ header:
     - 'LICENSE'
     - 'NOTICE'
     - '.gitignore'
+    - 'pkg/profiling/task/**/bpf_*'
```

Comment:
I think we should remove this? As codes are ignored from git already.

Reply:
Updated.